docker/Dockerfile-app

```diff
@@ -17,16 +17,23 @@ RUN yum install -y --disableplugin=subscription-manager rpm-devel
 # copy config
 COPY ./conf/app.conf /etc/ubi_manifest/app.conf
 
+ENV POETRY_VIRTUALENVS_IN_PROJECT=true
+
 COPY . .
 
+# install poetry
+RUN pip3 install pipx && pipx install poetry
```

Comment:
The base image already comes with prepared virtualenv it would good to re-use it and have a bit cleaner Dockerfiles.
Especially the CMD doesn't look super cool to me.

The poetry setup can be customized by settings various env. vars of config, I came across this real-life Dockerfile so maybe we can re-use some things https://github.com/wemake-services/wemake-django-template/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/docker/django/Dockerfile 
Also some discussion about that in https://github.com/orgs/python-poetry/discussions/1879

So overall - can you please check if we can re-use that existing virtualenv inside containers?

The
-  A) we shouldn't need to install pipx 
-  B) we dont' need to use `poetry run` every time 
-  C) the CMD can stay as it was before.

Reply:
Thanks! I have changed the dockerfiles so that Poetry uses the virtualenv that comes with the base image.